<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QGroupBox" name="profile_manager">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>Profile Manager</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignJustify|Qt::AlignVCenter</set>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <property name="spacing">
       <number>5</number>
      </property>
      <item row="2" column="0" colspan="2">
       <layout class="QGridLayout" name="profile_tools_layout">
        <item row="0" column="3">
         <widget class="QPushButton" name="delete_profile_button">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Delete Profile</string>
          </property>
          <property name="statusTip">
           <string/>
          </property>
          <property name="whatsThis">
           <string>Delete Profile</string>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton::disabled{background-color: rgb(210, 210, 210)}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="save_profile_button">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Save Profile</string>
          </property>
          <property name="statusTip">
           <string/>
          </property>
          <property name="whatsThis">
           <string>Save Profile</string>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton::disabled{background-color: rgb(210, 210, 210)}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="load_profile_button">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="toolTip">
           <string>Load Profile</string>
          </property>
          <property name="statusTip">
           <string/>
          </property>
          <property name="whatsThis">
           <string/>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton::disabled{background-color: rgb(210, 210, 210)}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="Line" name="line_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QComboBox" name="profile_chooser">
        <property name="toolTip">
         <string/>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="nucleic_acid_settings">
     <property name="title">
      <string>Nucleic Acid Settings</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <property name="horizontalSpacing">
       <number>5</number>
      </property>
      <property name="verticalSpacing">
       <number>6</number>
      </property>
      <item row="8" column="0" colspan="2">
       <layout class="QGridLayout" name="gridLayout_3">
        <property name="horizontalSpacing">
         <number>20</number>
        </property>
        <item row="3" column="1">
         <widget class="QDoubleSpinBox" name="theta_b">
          <property name="statusTip">
           <string/>
          </property>
          <property name="readOnly">
           <bool>false</bool>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="specialValueText">
           <string/>
          </property>
          <property name="accelerated">
           <bool>false</bool>
          </property>
          <property name="showGroupSeparator" stdset="0">
           <bool>false</bool>
          </property>
          <property name="suffix">
           <string notr="true">°</string>
          </property>
          <property name="decimals">
           <number>4</number>
          </property>
          <property name="maximum">
           <double>360.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.500000000000000</double>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QDoubleSpinBox" name="theta_c">
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="specialValueText">
           <string/>
          </property>
          <property name="accelerated">
           <bool>false</bool>
          </property>
          <property name="showGroupSeparator" stdset="0">
           <bool>false</bool>
          </property>
          <property name="suffix">
           <string notr="true">°</string>
          </property>
          <property name="decimals">
           <number>4</number>
          </property>
          <property name="maximum">
           <double>360.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.500000000000000</double>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QDoubleSpinBox" name="theta_s">
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="specialValueText">
           <string/>
          </property>
          <property name="accelerated">
           <bool>false</bool>
          </property>
          <property name="showGroupSeparator" stdset="0">
           <bool>false</bool>
          </property>
          <property name="prefix">
           <string/>
          </property>
          <property name="suffix">
           <string notr="true">°</string>
          </property>
          <property name="decimals">
           <number>4</number>
          </property>
          <property name="maximum">
           <double>360.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.500000000000000</double>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="theta_c_label">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="cursor">
           <cursorShape>ArrowCursor</cursorShape>
          </property>
          <property name="toolTip">
           <string/>
          </property>
          <property name="statusTip">
           <string/>
          </property>
          <property name="whatsThis">
           <string/>
          </property>
          <property name="text">
           <string>Θ&lt;sub&gt;c&lt;/sub&gt;</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="theta_s_label">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="cursor">
           <cursorShape>ArrowCursor</cursorShape>
          </property>
          <property name="toolTip">
           <string/>
          </property>
          <property name="statusTip">
           <string/>
          </property>
          <property name="whatsThis">
           <string/>
          </property>
          <property name="text">
           <string>Θ&lt;sub&gt;s&lt;/sub&gt;</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="theta_b_label">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="cursor">
           <cursorShape>ArrowCursor</cursorShape>
          </property>
          <property name="toolTip">
           <string/>
          </property>
          <property name="statusTip">
           <string/>
          </property>
          <property name="whatsThis">
           <string/>
          </property>
          <property name="text">
           <string>Θ&lt;sub&gt;b&lt;/sub&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="6" column="0" colspan="2">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="horizontalSpacing">
         <number>16</number>
        </property>
        <item row="2" column="0" colspan="2">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="topMargin">
           <number>5</number>
          </property>
          <property name="bottomMargin">
           <number>4</number>
          </property>
          <item>
           <widget class="QSpinBox" name="T">
            <property name="toolTip">
             <string>Turns</string>
            </property>
            <property name="whatsThis">
             <string/>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::NoButtons</enum>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>5</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="T_label">
            <property name="toolTip">
             <string/>
            </property>
            <property name="statusTip">
             <string/>
            </property>
            <property name="text">
             <string>[T]urns</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>5</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="bases_per_turns_slash">
            <property name="toolTip">
             <string>(per)</string>
            </property>
            <property name="statusTip">
             <string/>
            </property>
            <property name="text">
             <string>/</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>5</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QSpinBox" name="B">
            <property name="toolTip">
             <string>Bases</string>
            </property>
            <property name="statusTip">
             <string/>
            </property>
            <property name="whatsThis">
             <string/>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::NoButtons</enum>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>5</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="B_label">
            <property name="toolTip">
             <string>Bases</string>
            </property>
            <property name="statusTip">
             <string>There are B bases per T turns of the helix</string>
            </property>
            <property name="text">
             <string>[B]ases </string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="H_label">
          <property name="cursor">
           <cursorShape>ArrowCursor</cursorShape>
          </property>
          <property name="toolTip">
           <string>Helical Twist Height</string>
          </property>
          <property name="statusTip">
           <string/>
          </property>
          <property name="text">
           <string>[H]eight</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDoubleSpinBox" name="D">
          <property name="toolTip">
           <string/>
          </property>
          <property name="whatsThis">
           <string/>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="suffix">
           <string notr="true"> nm</string>
          </property>
          <property name="decimals">
           <number>4</number>
          </property>
          <property name="maximum">
           <double>999999.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.500000000000000</double>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="D_label">
          <property name="cursor">
           <cursorShape>ArrowCursor</cursorShape>
          </property>
          <property name="toolTip">
           <string/>
          </property>
          <property name="statusTip">
           <string/>
          </property>
          <property name="whatsThis">
           <string/>
          </property>
          <property name="text">
           <string>[D]iameter</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDoubleSpinBox" name="H">
          <property name="toolTip">
           <string/>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="suffix">
           <string notr="true"> nm</string>
          </property>
          <property name="decimals">
           <number>4</number>
          </property>
          <property name="maximum">
           <double>999999.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.500000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="Line" name="nucleic_acid_settings_splitter">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <layout class="QGridLayout" name="gridLayout_7">
        <property name="horizontalSpacing">
         <number>20</number>
        </property>
        <item row="2" column="1">
         <widget class="QDoubleSpinBox" name="Z_s">
          <property name="statusTip">
           <string/>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="suffix">
           <string notr="true"> nm</string>
          </property>
          <property name="decimals">
           <number>4</number>
          </property>
          <property name="maximum">
           <double>999999.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.500000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDoubleSpinBox" name="Z_c">
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="suffix">
           <string notr="true"> nm</string>
          </property>
          <property name="decimals">
           <number>4</number>
          </property>
          <property name="maximum">
           <double>999999.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.500000000000000</double>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="Z_s_label">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="cursor">
           <cursorShape>ArrowCursor</cursorShape>
          </property>
          <property name="toolTip">
           <string/>
          </property>
          <property name="statusTip">
           <string>The vertical height between two NEMids on different helices of the same double helix</string>
          </property>
          <property name="whatsThis">
           <string/>
          </property>
          <property name="text">
           <string>Z&lt;sub&gt;s&lt;/sub&gt;</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="Z_b_label">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="cursor">
           <cursorShape>ArrowCursor</cursorShape>
          </property>
          <property name="toolTip">
           <string/>
          </property>
          <property name="statusTip">
           <string/>
          </property>
          <property name="whatsThis">
           <string/>
          </property>
          <property name="text">
           <string>Z&lt;sub&gt;b&lt;/sub&gt;</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDoubleSpinBox" name="Z_b">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="statusTip">
           <string/>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">QDoubleSpinBox::disabled{background-color: rgb(240, 240, 240); color: black}</string>
          </property>
          <property name="wrapping">
           <bool>false</bool>
          </property>
          <property name="frame">
           <bool>true</bool>
          </property>
          <property name="readOnly">
           <bool>false</bool>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="specialValueText">
           <string/>
          </property>
          <property name="accelerated">
           <bool>false</bool>
          </property>
          <property name="showGroupSeparator" stdset="0">
           <bool>false</bool>
          </property>
          <property name="prefix">
           <string/>
          </property>
          <property name="suffix">
           <string notr="true"> nm</string>
          </property>
          <property name="decimals">
           <number>4</number>
          </property>
          <property name="minimum">
           <double>0.000000000000000</double>
          </property>
          <property name="maximum">
           <double>999999.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.500000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="Z_c_label">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="cursor">
           <cursorShape>ArrowCursor</cursorShape>
          </property>
          <property name="toolTip">
           <string/>
          </property>
          <property name="statusTip">
           <string/>
          </property>
          <property name="whatsThis">
           <string/>
          </property>
          <property name="text">
           <string>Z&lt;sub&gt;c&lt;/sub&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
